tests/compas_timber/test_wall.py

```diff
@@ -61,51 +60,40 @@ def test_add_elements_to_group(model, beam_list, wall1, wall2):
 
     assert model.has_group(wall1)
     assert model.has_group(wall2)
-    assert list(model.get_elements_in_group(wall1_group)) == [wall1, beam_a, beam_b]
-    assert list(model.get_elements_in_group(wall2_group)) == [wall2, beam_c, beam_d]
+    assert list(model.get_elements_in_group(wall1)) == [beam_a, beam_b]
+    assert list(model.get_elements_in_group(wall2)) == [beam_c, beam_d]
 
 
 def test_get_elements_in_group_filter(model, beam_list, wall1, wall2):
     beam_a, beam_b, beam_c, beam_d = beam_list
 
-    wall1_group = model.add_group_element(wall1)
-    wall2_group = model.add_group_element(wall2)
+    model.add_element(wall1)
+    model.add_element(wall2)
 
-    model.add_element(beam_a, parent=wall1_group)
-    model.add_element(beam_b, parent=wall1_group)
-    model.add_element(beam_c, parent=wall2_group)
-    model.add_element(beam_d, parent=wall2_group)
+    model.add_element(beam_a, parent=wall1)
+    model.add_element(beam_b, parent=wall1)
+    model.add_element(beam_c, parent=wall2)
+    model.add_element(beam_d, parent=wall2)
 
     assert model.has_group(wall1)
     assert model.has_group(wall2)
-    assert list(model.get_elements_in_group(wall1_group, filter_=lambda b: b.is_beam)) == [beam_a, beam_b]
-    assert list(model.get_elements_in_group(wall2_group, filter_=lambda b: b.is_beam)) == [beam_c, beam_d]
-    assert list(model.get_elements_in_group(wall1_group, filter_=lambda b: b.is_wall)) == [wall1]
-    assert list(model.get_elements_in_group(wall2_group, filter_=lambda b: b.is_wall)) == [wall2]
-    assert list(model.get_elements_in_group(wall1_group, filter_=lambda b: b.is_group_element)) == [wall1]
-    assert list(model.get_elements_in_group(wall2_group, filter_=lambda b: b.is_group_element)) == [wall2]
+    assert list(model.get_elements_in_group(wall1, filter_=lambda b: isinstance(b, Beam))) == [beam_a, beam_b]
+    assert list(model.get_elements_in_group(wall2, filter_=lambda b: isinstance(b, Beam))) == [beam_c, beam_d]
+    # NOTE: due to the new Tree structure, I got rid of some of the explicit Group functionality.
+    # Instead I add wall as normal element and used the parent argument when adding children to the wall.
+    # NOTE: changed to isinstance because is_beam, is_plate, are TimberElement properties
+    # NOTE: I am presuming we want to include various element types in groups, not just TimberElements and don't want to have to add is_* properties to all elements
```

Comment:
yea the `is_*` could potentially be retired because of the new `model.get_elemets_of_type(...)` but lets leave it to another PR.